migration/src/jira2github_import.py

```diff
@@ -123,6 +123,17 @@ def comment_author(author_name, author_dispname):
             author_gh = account_map.get(author_name)
             return f"{author_dispname} (@{author_gh})" if author_gh else author_dispname
         
+        def enable_hyperlink_to_commit(comment_body: str):
+            lines = []
+            for line in comment_body.split("\n"):
+                # remove '[' and ']' iff it contains a URL (i.e. link to a commit in ASF GitBox repo).
+                m = re.match(r"^\[\s?(https?://\S+)\s?\]$", line.strip())
```

Comment:
Maybe `\s*` instead of `\s?` after the opening `[` and before the closing `]` for better robustness?   Or are we sure it's always exactly 0 or 1 space?